pkgs/os-specific/linux/tuxedo-keyboard/default.nix

```
@@ -24,7 +24,13 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    description = "Full color keyboard driver for tuxedo computers laptops";
+    description = "Keyboard and hardware I/O driver for TUXEDO Computers laptops";
+    longDescription = ''
+      This driver provides support for Fn keys, brightness/color/mode for most TUXEDO
+      keyboards (except white backlight-only models).
+
+      Use the hardware.tuxedo-keyboard NixOS module to use this driver.
```

```suggestion
Can be used with the "hardware.tuxedo-keyboard" NixOS module.
```

Since strictly speaking the module and package distinct things and users could always just add 

```
      boot.kernelModules = ["tuxedo_keyboard"];
      boot.extraModulePackages = [ boot.kernelPackages.tuxedo-keyboard ];
```